daffodil-lib/src/main/resources/edu/illinois/ncsa/daffodil/xsd/built-in-formats.xsd

```diff
@@ -61,7 +61,7 @@
 				gets you only the new properties without all the property settings of daffodilTest1. -->
 
 			<dfdl:defineFormat name="daffodilNewProperties1">
-				<dfdl:format encodingErrorPolicy="replace" sequenceKind="ordered"
+				<dfdl:format encodingErrorPolicy="error" sequenceKind="ordered"
```

Comment:
Revert. We don't use this format. It was kept in for backward compat reasons. So we shouldn't be changing it.

Reply:
I'm a little confused about the errorEncodingPolicy stuff.  Are we changing most uses of it to "error", which we don't currently support and then relying on Daffodil switching it back to "replace" when the tests run?

Just want to make sure I am understanding what is going on.